### PR TITLE
solution

### PR DESCRIPTION
**Pull request by krolich500grn**
[DEMO LINK](https://krolich500grn.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.scss';
+
+export function Person({ name, sex, age, isMarried, partnerName }) {
```
Comment (jstmpelowycz) [inline src/components/Person/Person.jsx:6]:
```suggestion
export function Person({
  name,
  sex,
  age,
  isMarried,
  partnerName 
}) {
```

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age && `I am ${age}`}</p>
```
Comment (jstmpelowycz) [inline src/components/Person/Person.jsx:12]:
```suggestion
      {age &&
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      }
```

Comment (jstmpelowycz) [reply src/components/Person/Person.jsx:12]: I would recommend not rendering `p` and its content if no age provided

Comment (jstmpelowycz) [reply src/components/Person/Person.jsx:12]: Additionally, a small validation for a proper age

**Review by jstmpelowycz — APPROVED** [review]

File: src/App.js [hunk 3]
```diff
@@ -25,22 +25,9 @@ const alex = {
...
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">My husband&apos;s name is Maksym</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person {...misha} />
```
Comment (Ihor-Karpyn) [inline src/App.js:28]: Let's fix in all cases
```suggestion
    <Person person={misha} />
```

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**